examples/esm/index.js

```diff
@@ -1,7 +1,9 @@
 
 import { AlgorandTransactionCrafter } from '@algorandfoundation/algo-models'
+import {SignTransactionsParams} from "@algorandfoundation/algo-models/provider";
 
 let atc = new AlgorandTransactionCrafter('testnet-v1.0', 'SGO1GKSzyE7IEPItTxCByw9x8FmnrCDexi9/cOUJOiI=')
 let pay = atc.pay(123, 'R2HYOMCQIIVZLYYSVQ7RRDME5QB2NNACNIYEYIIN3NCTEKHEVE4RJRNWN4', 'R2HYOMCQIIVZLYYSVQ7RRDME5QB2NNACNIYEYIIN3NCTEKHEVE4RJRNWN4')
 //console.log(Buffer.from(pay.get().encode()).toString('base64'))
 console.log(pay)
+console.log(SignTransactionsParams)
```

Comment:
Is this file meant to be a test case?

Reply:
Exactly, it's for manual QA to ensure the three artifacts are produced properly.

- cjs uses /dist/cjs/*
- esm uses /dist/esm/*
- types/typescript uses /dist/types/*

Reply:
Can't they be through regular test suits? Someone importing index{.js, .ts, ) would run those bits by default, no?

Reply:
Absolutely is possible, although It's non trivial to test with Node.js/Browser/Typescript/package.json combinations. 

The examples folder is less than ideal but it's a good place to manually smoke test installing the package in different contexts until we have a test suite for the distributions.